### ISSUE TITLE
[1.15.2] Preconditions check failure during render

### ISSUE THREAD

**Issue by OreCruncher** [BluSunrize/ImmersiveEngineering#4276]
### Description of the issue:
Client crash when teleporting from nether to overworld.  The target chunk has some IE wiring.  If it makes a difference, I am playing multiplayer with a separate system for the server.

EDIT:  One possible unusual thing in my build is that I have a HV Wire Relay attached to a block that is just on the other side of a chunk boundary.

### Crashlog:
https://gist.github.com/OreCruncher/60cf47487251a374f44f7919b8fc3d10

Snip - complete stack in crash log:
```
java.lang.IllegalArgumentException: null
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:108) ~[guava-21.0.jar:?] {}
	at blusunrize.immersiveengineering.api.wires.utils.WireUtils.getVecForIICAt(WireUtils.java:210) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.api.ApiUtils.getVecForIICAt(ApiUtils.java:299) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.api.wires.Connection.generateCatenaryData(Connection.java:121) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.api.wires.ImmersiveConnectableTileEntity.genConnBlockstate(ImmersiveConnectableTileEntity.java:140) ~[?:?] {re:classloading}
	at blusunrize.immersiveengineering.api.wires.ImmersiveConnectableTileEntity.getModelData(ImmersiveConnectableTileEntity.java:153) ~[?:?] {re:classloading}
	at net.minecraftforge.client.model.ModelDataManager.refreshModelData(ModelDataManager.java:89) ~[?:?] {re:classloading}
	at net.minecraftforge.client.model.ModelDataManager.getModelData(ModelDataManager.java:117) ~[?:?] {re:classloading}
```

### Versions & Modlist
forge-1.15.2-31.2.36
ImmersiveEngineering-1.15.2-3.2.0-114
Mod List in crash report (All the Mods 5 2.18)

**Comment by BluSunrize** [comment]
Can you reproduce this on version 4.0.0-117 of ImmersiveEngineering?

**Comment by Shados** [comment]
I'm getting a client crash with essentially the same stack trace on version 1.15.2-4.0.1-119 (in a different modpack):
```
Stacktrace:
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:108)
	at blusunrize.immersiveengineering.api.wires.utils.WireUtils.getVecForIICAt(WireUtils.java:210)
	at blusunrize.immersiveengineering.api.ApiUtils.getVecForIICAt(ApiUtils.java:295)
	at blusunrize.immersiveengineering.api.wires.Connection.generateCatenaryData(Connection.java:121)
	at blusunrize.immersiveengineering.api.wires.ImmersiveConnectableTileEntity.genConnBlockstate(ImmersiveConnectableTileEntity.java:140)
	at blusunrize.immersiveengineering.api.wires.ImmersiveConnectableTileEntity.getModelData(ImmersiveConnectableTileEntity.java:153)
	at net.minecraftforge.client.model.ModelDataManager.refreshModelData(ModelDataManager.java:89)
	at net.minecraftforge.client.model.ModelDataManager.getModelData(ModelDataManager.java:117)
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender$ChunkRenderTask.<init>(ChunkRenderDispatcher.java:401)
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender$RebuildTask.<init>(ChunkRenderDispatcher.java:429)
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender.func_178574_d(ChunkRenderDispatcher.java:362)
	at net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$ChunkRender.func_228929_a_(ChunkRenderDispatcher.java:367)
	at net.minecraft.client.renderer.WorldRenderer.func_174967_a(WorldRenderer.java:1723)
	at net.minecraft.client.renderer.WorldRenderer.func_228426_a_(WorldRenderer.java:903)
	at net.minecraft.client.renderer.GameRenderer.func_228378_a_(GameRenderer.java:600)
```

It appears to happen infrequently when chunks with IE wiring load in visually.

**Comment by BluSunrize** [comment]
This issue should be resolved on the newest 1.15 release, 4.1.1
